packages/cadl-ranch-specs/http/parameters/spread/main.tsp

```diff
@@ -22,10 +22,10 @@ namespace Model {
   @scenarioDoc("""
   Test case for spread named model. 
   
-  Should generate request body model named `BodyParameter`.
+  Should not generate request body model named `BodyParameter`.
   Should generate an operation like below:
   ```
-  spreadAsRequestBody(bodyParameter: BodyParameter)
+  spreadAsRequestBody(name: string)
```

Comment:
I've changed the scenarioDoc, as now `op spreadAsRequestBody(...BodyParameter): NoContentResponse;` would be a spread.